CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 -   Fixed chip mixin to use correct accent color issue ([#72](https://github.com/etn-ccis/blui-angular-themes/issues/72)).
 -   Fixed `<blui-inline>` does not work with inline svg icons issue ([#133](https://github.com/etn-ccis/blui-angular-themes/issues/133)).
 -   Fixed Dark theme `<blui-drawer-header>` text and icon color issue ([#79](https://github.com/etn-ccis/blui-angular-themes/issues/79)).
+-   Fixed Remove Edge show password icon for Blui-angular-workflow repo issue ([#137](https://github.com/etn-ccis/blui-angular-workflows/issues/137)).
```

Comment:
We should not include repo names in the changelog. I have transferred the issue to blui-angular-themes repo, please update the link to https://github.com/etn-ccis/blui-angular-themes/issues/168.

Also statement should be somewhat similar to https://github.com/etn-ccis/blui-react-workflows/blob/c7df2557ac8199b139b45b242fa032f2b8c4bb22/login-workflow/CHANGELOG.md?plain=1#L16

Reply:
To Suraj's point, please read the changelog item after you finish writing it. These bullet points should read like a sentence, not a copy & paste from GitHub.